### DISCUSSION TITLE
Autocomplete `ComboboxItem` instead of `ComboboxStringItem`

### DISCUSSION THREAD

**Discussion by engin-can** [mantinedev/mantine#6481] [Q&A]
I noticed `renderOption` complains about `label` since it's missing in `ComboboxStringItem`.  Can we change this to `ComboboxItem` instead?

**Code**
```
  const renderOption: AutocompleteProps['renderOption'] = ({ option }) => (
    <a href="#" target="_blank" rel="noreferrer">
      {option.label}
    </a>
  );
```

**Error**
`Property 'label' does not exist on type 'ComboboxStringItem'.`

**Source code**
https://github.com/mantinedev/mantine/blob/adadb00a3f5db75b59c84b1317f19aa79a1e44f3/packages/%40mantine/core/src/components/Autocomplete/Autocomplete.tsx#L49C1-L50C1

```
/** A function to render content of the option, replaces the default content of the option */
renderOption?: (input: ComboboxLikeRenderOptionInput<ComboboxStringItem>) => React.ReactNode;
```

**Comment by rtivital** [comment]
Autocomplete does not support label/value data format – https://mantine.dev/core/autocomplete/#data-formats
label and value are always the same. There is no reason to change this logic.
See also – https://help.mantine.dev/q/select-autocomplete-difference
